### DISCUSSION TITLE
[question] [pcre2] Turn off the jit for osx arm.

### DISCUSSION THREAD

**Discussion by talregev** [microsoft/vcpkg#36386] [Q&A] [answered]
I am using macosx-version-min=10.13.
In this version on arm i got compilation error for jit compilation. The feature is always on.
How can I turn it off? When I install it with wxwidgets it always on by definition. 
Can we add it turn off for osx arm? Maybe other solution, other then copy the port and create a custom port? 

Thank you for your help.

**Comment by dg0yt** [comment, marked as answer]
The preferred way is to only install `pcre2[core]`. 

But this solution might be spoiled by reverse dependencies lacking `"default-features": false`. These ports should be fixed.

As a mitigation, you could customize your triplet:
~~~cmake
if(PORT STREQUAL "pcre2")
    set(VCPKG_CMAKE_CONFIGURE_OPTIONS -DPCRE2_SUPPORT_JIT=OFF)
endif()
~~~
This should override the effect of this particular port feature.

**Comment by talregev** [comment]
Thank you! :)
